app/controllers/admin/semester_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Admin::SemesterController < Admin::BaseController
+  load_and_authorize_resource
+
+  def new
+  end
+
+  def create
+    if @semester.save
+      flash[:info] = "Created semester."
+      current_user.add_role_for_semester(Role::INSTRUCTOR, @semester)
+      redirect_to admin_semester_path(@semester)
+    else
+      flash[:error] = "Problem creating semester."
+      render "new"
+    end
+  end
+
+  def show
+  end
+
+  private
+
+  def semester_params
+    params.require(:semester).permit(:semester, :year, :lower_division_limit, :upper_division_limit, :application_deadline, :acceptance_release_date)
```

Comment:
Style: Break this to multiple lines.  We want to keep each line under 100 characters

```ruby
params.require(:semester).permit(
    :semester, :year, :lower_division_limit, :upper_division_limit,
    :application_deadline, :acceptance_release_date
)
```